### DISCUSSION TITLE
Unable to process my application for student developer pack benefits.

### DISCUSSION THREAD

**Discussion by 0x-buffer** [community/community#73124] [GitHub Education]
### Hi everyone,

This is my second time activating the developer benefits which was given to me by my college. While registering for the first time i has submitted my semster registration slip which was accepted instantly. But now when submitting the registration slip for my new semester it is giving me and error saying :

Your application cannot be reviewed until you fix the following:
• Your uploaded proof has already been used previously.

So then I tried submitting the ID Card but then its throwing me and error saying:

Unfortunately, we were unable to verify your academic affiliation based on the information you submitted on Nov 1, 2023

So what should I Do!

**Comment by williamregal** [comment]
The [Amazon Web Services DVA-C02 Exam Objectives](https://www.examsvce.com/DVA-C02-exam.html) that evaluates your knowledge and skills in developing applications on AWS. It covers various domains, each with its set of objectives. To pass this exam, you must demonstrate your proficiency in designing, developing, and deploying applications on AWS, as well as your understanding of best practices and security measures.

**Comment by YelchaninovArtem** [comment]
I have the same problem

**Comment by CrossoverRed** [comment]
I had the same problem, but the email they sent on why the application was not accepted as the image of student card without saying valid until when will cause it to not accept, another factor was not having a 2-Factor Verification which I had to do too. The way that worked and was able to submit the application is a screenshot in the computer (Window+Shift+S) of the official certificate of being a student in the school pdf file that shows I am a student until when.

**Comment by kiterza** [comment]
I have a similar problem, but his tip is that the email has been used for the application, but my application has expired

**Reply by kiterza** [reply]
@samus-aran  help me！

**Comment by sarahbstein** [comment]
I'm having a similar issue, too!
